presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -0,0 +1,106 @@
+================================
+Built-in System Access Control
+================================
+
+A system access control plugin enforces authorization at a global level,
+before any connector level authorization. You can either use one of the built-in
+plugins in Presto or provide your own by following the guidelines in
+:doc:`/develop/system-access-control`. Presto offers three built-in plugins:
+
+================================================== ============================================================
+Plugin Name                                        Description
+================================================== ============================================================
+``allow-all`` (default value)                      All operations are permitted.
+
+``read-only``                                      Operations that read data or metadata are permitted, but
+                                                   none of the operations that write data or metadata, are
+                                                   allowed. See :ref:`read-only-system-access-control` for
+                                                   details.
+
+``file``                                           Authorization checks are enforced using a config file specified
+                                                   by the configuration property ``security.config-file``. See
+                                                   :ref:`file-based-system-access-control` for details.
+================================================== ============================================================
+
+.. _allow-all-system-access-control:
+
+Allow All System Access Control
+===============================
+
+All operations are permitted under this plugin. This plugin is enabled by default.
+
+.. _read-only-system-access-control:
```

Comment:
sphinx failed to  build when referring the headers. So I reverted back the changes.